clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -0,0 +1,266 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+TODO: This measurement should replace api_responsiveness.go.
+*/
+
+package slos
+
+import (
+	"fmt"
+	"math"
+	"sort"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/common/model"
+	"k8s.io/apimachinery/pkg/util/sets"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+	"k8s.io/perf-tests/clusterloader2/pkg/errors"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	apiResponsivenessPrometheusMeasurementName = "APIResponsivenessPrometheus"
+
+	latencyQuery = "quantile_over_time(0.99, apiserver:apiserver_request_latency:histogram_quantile[%v])"
+	countQuery   = "sum(increase(apiserver_request_latency_seconds_count[%v])) by (resource, subresource, scope, verb)"
+
+	latencyWindowSize = 5 * time.Minute
+)
+
+func init() {
+	measurement.Register(apiResponsivenessPrometheusMeasurementName, createAPIResponsivenessPrometheusMeasurement)
```

Comment:
What are the official guidelines regarding the line length? 
Looks like you're exceeding the github suggested line length in many places.

Reply:
> Looks like you're exceeding the github suggested line length in many places.

TBH I'm not sure what is github suggested line length... Where can I find it?

Reply:
Probably there is none. Let's leave it as is :)